src/com/intellij/plugins/thrift/lang/psi/ThriftTypeReference.java

```diff
@@ -88,4 +90,10 @@ private boolean isSimple() {
       return fun.fun(Pair.create(name, getElement().getContainingFile()));
     }
   }
+
+  @NotNull
+  @Override
+  public Collection<? extends SymbolResolveResult> resolveReference() {
+    return super.resolveReference();
```

Comment:
is this required if it only calls `super. resolveReference()`?

Reply:
This is required because I get a compile error if not added.
`Error:(26, 8) java: com.intellij.plugins.thrift.lang.psi.ThriftTypeReference is not abstract and does not override abstract method resolveReference() in com.intellij.model.SymbolReference`

It should resolve to https://github.com/JetBrains/intellij-community/blob/07499d66aad55b7e3d66473448ca2afde296ec82/platform/core-api/src/com/intellij/psi/PsiReference.java#L138

I tried to debug it by adding breakpoint here, but wasn't able to hit it to check what `super.resolveReference()` resolves to.